### ISSUE TITLE
ygg torrent can't be add on prowlar or jackett

### ISSUE THREAD

**Issue by thegrunge36** [FlareSolverr/FlareSolverr#972]
### Have you checked our README?

- [X] I have checked the README

### Have you followed our Troubleshooting?

- [X] I have followed your Troubleshooting

### Is there already an issue for your problem?

- [X] I have checked older issues, open and closed

### Have you checked the discussions?

- [X] I have read the Discussions

### Environment

```markdown
- FlareSolverr version: 3.3.10
- Last working FlareSolverr version:
- Operating system:Debian 12
- Are you using Docker: [yes/no] no
- FlareSolverr User-Agent (see log traces or / endpoint):
- Are you using a VPN: [yes/no] yes
- Are you using a Proxy: [yes/no] no
- Are you using Captcha Solver: [yes/no] no
- If using captcha solver, which one:
- URL to test this issue:
```

### Description

Hello, for about 3 weeks I have had this error message when I try to add ygg. When I go by cookie it works but not for long. Otherwise by torrent I have this error. I tried different version of flaresolverr.

Unable to connect to indexer, indexer's server is unavailable. Try again later. HTTP request failed: [500:InternalServerError] [POST] at [http://localhost:8191/v1]

flaresolverr.service - flaresolverr
     Loaded: loaded (/etc/systemd/system/flaresolverr.service; enabled; preset: enabled)
     Active: active (running) since Wed 2023-11-15 13:04:52 EST; 19h ago
   Main PID: 45604 (flaresolverr)
      Tasks: 6 (limit: 38207)
     Memory: 210.5M
        CPU: 4min 22.897s
     CGroup: /system.slice/flaresolverr.service
             ├─45604 /home/jonathan/flaresolverr/flaresolverr
             └─45629 Xvfb :151124646 -screen 0 800x680x24

nov 15 15:14:45 jonathan flaresolverr[45604]: 2023-11-15 15:14:45 INFO     Response in 0.797 s
nov 15 15:14:45 jonathan flaresolverr[45604]: 2023-11-15 15:14:45 INFO     127.0.0.1 POST http://localhost:8191/v1 200 OK
nov 15 21:15:16 jonathan flaresolverr[45604]: 2023-11-15 21:15:16 INFO     Incoming request => POST /v1 body: {'maxTimeout': 60000, 'cmd': 'request.get', 'url': 'https://prowlarr.servarr.com/v1/ping', 'proxy': >
nov 15 21:15:16 jonathan flaresolverr[45604]: 2023-11-15 21:15:16 INFO     Challenge not detected!
nov 15 21:15:16 jonathan flaresolverr[45604]: 2023-11-15 21:15:16 INFO     Response in 0.817 s
nov 15 21:15:16 jonathan flaresolverr[45604]: 2023-11-15 21:15:16 INFO     127.0.0.1 POST http://localhost:8191/v1 200 OK
nov 16 03:15:42 jonathan flaresolverr[45604]: 2023-11-16 03:15:42 INFO     Incoming request => POST /v1 body: {'maxTimeout': 60000, 'cmd': 'request.get', 'url': 'https://prowlarr.servarr.com/v1/ping', 'proxy': >
nov 16 03:15:43 jonathan flaresolverr[45604]: 2023-11-16 03:15:43 INFO     Challenge not detected!
nov 16 03:15:43 jonathan flaresolverr[45604]: 2023-11-16 03:15:43 INFO     Response in 0.858 s
nov 16 03:15:43 jonathan flaresolverr[45604]: 2023-11-16 03:15:43 INFO     127.0.0.1 POST http://localhost:8191/v1 200 OK

### Logged Error Messages

```text
2023-11-15 08:01:40.9|Warn|ProwlarrErrorPipeline|Invalid request Validation failed: 
 -- : Unable to connect to indexer, indexer's server is unavailable. Try again later. HTTP request failed: [500:InternalServerError] [POST] at [http://localhost:8191/v1]
2023-11-15 08:02:00.6|Info|Auth|Auth-Success ip ::1 username
```

**Comment by Compr0mzd** [comment]
Got the issue as well nowadays

**Comment by ilike2burnthing** [comment]
https://github.com/FlareSolverr/FlareSolverr/issues/951#issuecomment-1809524306

**Comment by thegrunge36** [comment]
Even with this configuration i have this result

[Unit]
Description=flaresolverr
After=network-online.target

[Service]
SyslogIdentifier=flaresolverr
Restart=always
RestartSec=5
Type=simple
User=jonathan
Group=jonathan
Environment="LOG_LEVEL=info"
Environment="CAPTCHA_SOLVER=none"
WorkingDirectory=/home/jonathan
ExecStart=/home/jonathan/flaresolverr/flaresolverr
TimeoutStopSec=30

[Install]
WantedBy=multi-user.target


flaresolverr.service - flaresolverr
     Loaded: loaded (/etc/systemd/system/flaresolverr.service; enabled; preset: enabled)
     Active: active (running) since Fri 2023-11-17 08:06:12 EST; 2min 3s ago
   Main PID: 224198 (flaresolverr)
      Tasks: 130 (limit: 38207)
     Memory: 541.7M
        CPU: 19.543s
     CGroup: /system.slice/flaresolverr.service
             ├─224198 /home/jonathan/flaresolverr/flaresolverr
             ├─224226 Xvfb :1673848418 -screen 0 800x680x24
             ├─224936 /home/jonathan/flaresolverr/chrome/chrome --no-sandbox --window-size=1920,1080 --disable-setuid-sandbox --disable-dev-shm-usage --no-zygote --disable-gpu-sandbox --disable-software-rasteri>
             ├─224937 /home/jonathan/.local/share/undetected_chromedriver/chromedriver --port=46789
             ├─224943 /home/jonathan/flaresolverr/chrome/chrome_crashpad_handler --monitor-self --monitor-self-annotation=ptype=crashpad-handler "--database=/home/jonathan/.config/chromium/Crash Reports" "--ann>
             ├─224945 /home/jonathan/flaresolverr/chrome/chrome_crashpad_handler --no-periodic-tasks --monitor-self-annotation=ptype=crashpad-handler "--database=/home/jonathan/.config/chromium/Crash Reports" ">
             ├─224969 "/home/jonathan/flaresolverr/chrome/chrome --type=utility --utility-sub-type=network.mojom.NetworkService --lang=fr --service-sandbox-type=network --no-sandbox --disable-dev-shm-usage --ig>
             ├─224970 "/home/jonathan/flaresolverr/chrome/chrome --type=utility --utility-sub-type=storage.mojom.StorageService --lang=fr --service-sandbox-type=utility --no-sandbox --disable-dev-shm-usage --ig>
             ├─225040 "/home/jonathan/flaresolverr/chrome/chrome --type=gpu-process --disable-gpu-sandbox --no-sandbox --disable-dev-shm-usage --log-level=0 --crashpad-handler-pid=224943 --enable-crash-reporter>
             ├─225076 "/home/jonathan/flaresolverr/chrome/chrome --type=renderer --crashpad-handler-pid=224943 --enable-crash-reporter=, --user-agent=Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, l>
             ├─225111 "/home/jonathan/flaresolverr/chrome/chrome --type=renderer --crashpad-handler-pid=224943 --enable-crash-reporter=, --user-agent=Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, l>
             └─225183 "/home/jonathan/flaresolverr/chrome/chrome --type=renderer --crashpad-handler-pid=224943 --enable-crash-reporter=, --user-agent=Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, l>

nov 17 08:06:19 jonathan flaresolverr[224198]: 2023-11-17 08:06:19 INFO     Test successful!
nov 17 08:06:19 jonathan flaresolverr[224198]: 2023-11-17 08:06:19 INFO     Serving on http://0.0.0.0:8191
nov 17 08:06:45 jonathan flaresolverr[224198]: 2023-11-17 08:06:45 INFO     127.0.0.1 GET http://localhost:8191/ 200 OK
nov 17 08:07:12 jonathan flaresolverr[224198]: 2023-11-17 08:07:12 INFO     Incoming request => POST /v1 body: {'maxTimeout': 60000, 'cmd': 'request.get', 'url': 'https://www3.yggtorrent.wtf/engine/search?categ>
nov 17 08:07:12 jonathan flaresolverr[224198]: 2023-11-17 08:07:12 INFO     Challenge detected. Selector found: #challenge-spinner
nov 17 08:08:04 jonathan flaresolverr[224198]: 2023-11-17 08:08:04 INFO     Incoming request => POST /v1 body: {'maxTimeout': 60000, 'cmd': 'request.get', 'url': 'https://www3.yggtorrent.wtf/engine/search?categ>
nov 17 08:08:05 jonathan flaresolverr[224198]: 2023-11-17 08:08:05 INFO     Challenge detected. Selector found: #challenge-spinner
nov 17 08:08:12 jonathan flaresolverr[224198]: 2023-11-17 08:08:12 ERROR    Error: Error solving the challenge. Timeout after 60.0 seconds.
nov 17 08:08:12 jonathan flaresolverr[224198]: 2023-11-17 08:08:12 INFO     Response in 60.723 s
nov 17 08:08:12 jonathan flaresolverr[224198]: 2023-11-17 08:08:12 INFO     127.0.0.1 POST http://localhost:8191/v1 500 Internal Server Error

**Comment by ilike2burnthing** [comment]
If your system language is already French, then `--lang=fr` isn't changing anything. You need to use an English language code to bypass Cloudflare for YGG, as in the example.

**Comment by thegrunge36** [comment]
> If your system language is already French, then `--lang=fr` isn't changing anything. You need to use an English language code to bypass Cloudflare for YGG, as in the example.

Is this the right way to do it? because even like that it doesn't work.
Thank you!

[Unit]
Description=flaresolverr
After=network-online.target

[Service]
SyslogIdentifier=flaresolverr
Restart=always
RestartSec=5
Type=simple
User=jonathan
Group=jonathan
Environment="LOG_LEVEL=info"
Environment="CAPTCHA_SOLVER=none"
Environment="LANG=en_GB"
WorkingDirectory=/home/jonathan
ExecStart=/home/jonathan/flaresolverr/flaresolverr
TimeoutStopSec=30

[Install]
WantedBy=multi-user.target

**Comment by thegrunge36** [comment]
I installed the latest version with Docker, it works this way rather than the traditional way. But I would still like how I would have gone about settling with the .service

**Comment by ilike2burnthing** [comment]
I'm not familiar with unit files, but as far as I can tell from [systemd.exec#Environment](https://www.freedesktop.org/software/systemd/man/latest/systemd.exec.html#Environment) that looks correct. There shouldn't be any difference, but you could also try:

> Environment=LOG_LEVEL=info CAPTCHA_SOLVER=none LANG=en_GB

If you access FlareSolverr in browser from the same device (e.g. `http://127.0.0.1:8191`), what is the response?

**Comment by thegrunge36** [comment]
> http://127.0.0.1:8191

Thanks, I'll try this way to see if it works too. Whether I access with locahost:8191 or http://127.0.0.1:8191 it works

**Comment by ilike2burnthing** [comment]
Sorry, I meant please provide the response, rather than just checking if it was working.

**Comment by thegrunge36** [comment]
> Sorry, I meant please provide the response, rather than just checking if it was working.

I tried with these settings, but it doesn't work with the .service. I will continue with docker.

[Unit]
Description=flaresolverr
After=network-online.target

[Service]
SyslogIdentifier=flaresolverr
Restart=always
RestartSec=5
Type=simple
User=jonathan
Group=jonathan
Environment=LOG_LEVEL=info
Environment=LOG_LEVEL=info CAPTCHA_SOLVER=none LANG=en_GB
WorkingDirectory=/home/jonathan
ExecStart=/home/jonathan/flaresolverr/flaresolverr
TimeoutStopSec=30

[Install]
WantedBy=multi-user.target



flaresolverr.service - flaresolverr
     Loaded: loaded (/etc/systemd/system/flaresolverr.service; disabled; preset: enabled)
     Active: active (running) since Fri 2023-11-17 16:12:26 EST; 4min 21s ago
   Main PID: 273734 (flaresolverr)
      Tasks: 6 (limit: 38207)
     Memory: 370.6M
        CPU: 16.084s
     CGroup: /system.slice/flaresolverr.service
             ├─273734 /home/jonathan/flaresolverr/flaresolverr
             └─273760 Xvfb :221094603 -screen 0 800x680x24

nov 17 16:12:26 jonathan flaresolverr[273734]: 2023-11-17 16:12:26 INFO     Launching web browser...
nov 17 16:12:28 jonathan flaresolverr[273734]: 2023-11-17 16:12:28 INFO     FlareSolverr User-Agent: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/115.0.0.0 Safari/537.36
nov 17 16:12:28 jonathan flaresolverr[273734]: 2023-11-17 16:12:28 INFO     Test successful!
nov 17 16:12:28 jonathan flaresolverr[273734]: 2023-11-17 16:12:28 INFO     Serving on http://0.0.0.0:8191
nov 17 16:12:46 jonathan flaresolverr[273734]: 2023-11-17 16:12:46 INFO     127.0.0.1 GET http://127.0.0.1:8191/ 200 OK
nov 17 16:13:26 jonathan flaresolverr[273734]: 2023-11-17 16:13:26 INFO     Incoming request => POST /v1 body: {'maxTimeout': 60000, 'cmd': 'request.get', 'url': 'https://www3.yggtorrent.wtf/engine/search?categ>
nov 17 16:13:27 jonathan flaresolverr[273734]: 2023-11-17 16:13:27 INFO     Challenge detected. Selector found: #challenge-spinner
nov 17 16:14:27 jonathan flaresolverr[273734]: 2023-11-17 16:14:27 ERROR    Error: Error solving the challenge. Timeout after 60.0 seconds.
nov 17 16:14:27 jonathan flaresolverr[273734]: 2023-11-17 16:14:27 INFO     Response in 60.743 s
nov 17 16:14:27 jonathan flaresolverr[273734]: 2023-11-17 16:14:27 INFO     127.0.0.1 POST http://localhost:8191/v1 500 Internal Server Error

**Comment by ilike2burnthing** [comment]
Remove the first `Environment=LOG_LEVEL=info` line in case it's causing an issue. You could also try changing the log level to `debug` to see if you then get debug logs. If not, there's a bigger issue.

I just need to confirm you're definitely running v3.3.10, so please provide the in browser response to accessing FlareSolverr's API endpoint.

**Comment by thegrunge36** [comment]
> Remove the first `Environment=LOG_LEVEL=info` line in case it's causing an issue. You could also try changing the log level to `debug` to see if you then get debug logs. If not, there's a bigger issue.
> 
> I just need to confirm you're definitely running v3.3.10, so please provide the in browser response to accessing FlareSolverr's API endpoint.

I try with this and yes i have 3.3.10 version 

[Unit]
Description=flaresolverr
After=network-online.target

[Service]
SyslogIdentifier=flaresolverr
Restart=always
RestartSec=5
Type=simple
User=jonathan
Group=jonathan
Environment=LOG_LEVEL=debug CAPTCHA_SOLVER=none LANG=en_GB
WorkingDirectory=/home/jonathan
ExecStart=/home/jonathan/flaresolverr/flaresolverr
TimeoutStopSec=30

[Install]
WantedBy=multi-user.target



flaresolverr.service - flaresolverr
     Loaded: loaded (/etc/systemd/system/flaresolverr.service; disabled; preset: enabled)
     Active: active (running) since Fri 2023-11-17 16:25:55 EST; 2min 55s ago
   Main PID: 279515 (flaresolverr)
      Tasks: 6 (limit: 38207)
     Memory: 146.5M
        CPU: 16.967s
     CGroup: /system.slice/flaresolverr.service
             ├─279515 /home/jonathan/flaresolverr/flaresolverr
             └─279542 Xvfb :6053107 -screen 0 800x680x24

nov 17 16:27:42 jonathan flaresolverr[279515]: 2023-11-17 16:27:42 DEBUG    ReqId 139923969525440 Timeout waiting for selector
nov 17 16:27:42 jonathan flaresolverr[279515]: 2023-11-17 16:27:42 DEBUG    ReqId 139923969525440 Try to find the Cloudflare verify checkbox...
nov 17 16:27:42 jonathan flaresolverr[279515]: 2023-11-17 16:27:42 DEBUG    ReqId 139923969525440 Cloudflare verify checkbox not found on the page.
nov 17 16:27:42 jonathan flaresolverr[279515]: 2023-11-17 16:27:42 DEBUG    ReqId 139923969525440 Try to find the Cloudflare 'Verify you are human' button...
nov 17 16:27:42 jonathan flaresolverr[279515]: 2023-11-17 16:27:42 DEBUG    ReqId 139923969525440 The Cloudflare 'Verify you are human' button not found on the page.
nov 17 16:27:43 jonathan flaresolverr[279515]: 2023-11-17 16:27:43 DEBUG    ReqId 139923994703552 A used instance of webdriver has been destroyed
nov 17 16:27:43 jonathan flaresolverr[279515]: 2023-11-17 16:27:43 ERROR    ReqId 139923994703552 Error: Error solving the challenge. Timeout after 60.0 seconds.
nov 17 16:27:43 jonathan flaresolverr[279515]: 2023-11-17 16:27:43 DEBUG    ReqId 139923994703552 Response => POST /v1 body: {'status': 'error', 'message': 'Error: Error solving the challenge. Timeout after 60.>
nov 17 16:27:43 jonathan flaresolverr[279515]: 2023-11-17 16:27:43 INFO     ReqId 139923994703552 Response in 60.725 s
nov 17 16:27:43 jonathan flaresolverr[279515]: 2023-11-17 16:27:43 INFO     ReqId 139923994703552 127.0.0.1 POST http://localhost:8191/v1 500 Internal Server Error

**Comment by ilike2burnthing** [comment]
Again, please provide the in browser response to accessing FlareSolverr's API endpoint.

At least we know `Environment` works then, as you're getting debug logs.

Are you using your VPN with the Docker version of FlareSolverr?

**Comment by thegrunge36** [comment]
> Again, please provide the in browser response to accessing FlareSolverr's API endpoint.
> 
> At least we know `Environment` works then, as you're getting debug logs.
> 
> Are you using your VPN with the Docker version of FlareSolverr?

I don't use VPN either for the browser or with docker. where can I find the in browser response to accessing FlareSolverr's API endpoint?

**Comment by thegrunge36** [comment]
> Again, please provide the in browser response to accessing FlareSolverr's API endpoint.
> 
> At least we know `Environment` works then, as you're getting debug logs.
> 
> Are you using your VPN with the Docker version of FlareSolverr?

Whether with docker or the .service, it works. It's only when I want to add ygg that it doesn't work with the .service.

**Comment by ilike2burnthing** [comment]
Add the environment variable `HEADLESS=false`, see if it shows the Cloudflare challenge in English. If so, remove `LANG=en_GB`; does it now show the challenge in French? If so, add `LANG` again but this time with the value `de_DE`; is it now German?

**Comment by thegrunge36** [comment]
> HEADLESS=false

it's stuck there

nov 17 17:29:13 jonathan flaresolverr[287660]: 2023-11-17 17:29:13 DEBUG    ReqId 139786834571328 Started executable: `/home/jonathan/.local/share/undetected_chromedriver/undetected_chromedriver` in a child process with pid: 287952

**Comment by ilike2burnthing** [comment]
Well that's not great. How long did you leave it for?

Remove headless and lang for now. Test with the indexers iDope, TorrentQQ, and Torrent[CORE]; do they work?

**Comment by thegrunge36** [comment]
> > HEADLESS=false
> 
> it's stuck there
> 
> nov 17 17:31:32 jonathan flaresolverr[288054]: 2023-11-17 17:31:32 INFO ReqId 140256244809792 FlareSolverr 3.3.10 nov 17 17:31:32 jonathan flaresolverr[288054]: 2023-11-17 17:31:32 DEBUG ReqId 140256244809792 Debug log enabled nov 17 17:31:32 jonathan flaresolverr[288054]: 2023-11-17 17:31:32 INFO ReqId 140256244809792 Testing web browser installation... nov 17 17:31:32 jonathan flaresolverr[288054]: 2023-11-17 17:31:32 INFO ReqId 140256244809792 Platform: Linux-6.1.0-13-amd64-x86_64-with-glibc2.36 nov 17 17:31:32 jonathan flaresolverr[288054]: 2023-11-17 17:31:32 INFO ReqId 140256244809792 Chrome / Chromium path: /home/jonathan/flaresolverr/chrome/chrome nov 17 17:31:32 jonathan flaresolverr[288054]: 2023-11-17 17:31:32 INFO ReqId 140256244809792 Chrome / Chromium major version: 115 nov 17 17:31:32 jonathan flaresolverr[288054]: 2023-11-17 17:31:32 INFO ReqId 140256244809792 Launching web browser... nov 17 17:31:32 jonathan flaresolverr[288054]: 2023-11-17 17:31:32 DEBUG ReqId 140256244809792 Launching web browser... nov 17 17:29:13 jonathan flaresolverr[287660]: 2023-11-17 17:29:13 DEBUG ReqId 139786834571328 Started executable: `/home/jonathan/.local/share/undetected_chromedriver/undetected_chromedriver` in a child process with pid: 287952



> Well that's not great. How long did you leave it for?

It hasn't worked for about 2 weeks, but with Docker yes. Maybe it's a system problem too who knows, I don't have another site to test flaresolverr apart from ygg

**Comment by ilike2burnthing** [comment]
If you're using Jackett or Prowlarr:

> Test with the indexers iDope, TorrentQQ, and Torrent[CORE]

Otherwise:
- https://idope.se/
- https://torrentqq283.com/
- https://torrentcore.xyz/

**Comment by thegrunge36** [comment]
> If you're using Jackett or Prowlarr:
> 
> > Test with the indexers iDope, TorrentQQ, and Torrent[CORE]
> 
> Otherwise:
> 
> * https://idope.se/
> * https://torrentqq283.com/
> * https://torrentcore.xyz/

idope ended up working we can see the difference, https://torrentqq283.com/ does not have the right address in prowlarr and torrentcore.xyz is not in prowlarr.

flaresolverr.service - flaresolverr
     Loaded: loaded (/etc/systemd/system/flaresolverr.service; disabled; preset: enabled)
     Active: active (running) since Sat 2023-11-18 09:20:54 EST; 1min 31s ago
   Main PID: 336328 (flaresolverr)
      Tasks: 125 (limit: 38207)
     Memory: 549.2M
        CPU: 16.942s
     CGroup: /system.slice/flaresolverr.service
             ├─336328 /home/jonathan/flaresolverr/flaresolverr
             ├─336333 Xvfb :970390971 -screen 0 800x680x24
             ├─336875 /home/jonathan/flaresolverr/chrome/chrome --no-sandbox --window-size=1920,1080 --disable-setuid-sandbox --disable-dev-shm-usage --no-zygote --disable-gpu-sandbox --disable-software-rasteri>
             ├─336876 /home/jonathan/.local/share/undetected_chromedriver/chromedriver --port=39623
             ├─336882 /home/jonathan/flaresolverr/chrome/chrome_crashpad_handler --monitor-self --monitor-self-annotation=ptype=crashpad-handler "--database=/home/jonathan/.config/chromium/Crash Reports" "--ann>
             ├─336884 /home/jonathan/flaresolverr/chrome/chrome_crashpad_handler --no-periodic-tasks --monitor-self-annotation=ptype=crashpad-handler "--database=/home/jonathan/.config/chromium/Crash Reports" ">
             ├─336908 "/home/jonathan/flaresolverr/chrome/chrome --type=utility --utility-sub-type=network.mojom.NetworkService --lang=fr --service-sandbox-type=network --no-sandbox --disable-dev-shm-usage --ig>
             ├─336909 "/home/jonathan/flaresolverr/chrome/chrome --type=utility --utility-sub-type=storage.mojom.StorageService --lang=fr --service-sandbox-type=utility --no-sandbox --disable-dev-shm-usage --ig>
             ├─336980 "/home/jonathan/flaresolverr/chrome/chrome --type=gpu-process --disable-gpu-sandbox --no-sandbox --disable-dev-shm-usage --log-level=0 --crashpad-handler-pid=336882 --enable-crash-reporter>
             ├─337015 "/home/jonathan/flaresolverr/chrome/chrome --type=renderer --crashpad-handler-pid=336882 --enable-crash-reporter=, --user-agent=Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, l>
             ├─337040 "/home/jonathan/flaresolverr/chrome/chrome --type=renderer --crashpad-handler-pid=336882 --enable-crash-reporter=, --user-agent=Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, l>
             ├─337050 "/home/jonathan/flaresolverr/chrome/chrome --type=renderer --crashpad-handler-pid=336882 --enable-crash-reporter=, --user-agent=Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, l>
             └─337062 "/home/jonathan/flaresolverr/chrome/chrome --type=utility --utility-sub-type=data_decoder.mojom.DataDecoderService --lang=fr --service-sandbox-type=service --no-sandbox --disable-dev-shm-u>

nov 18 09:22:12 jonathan flaresolverr[336328]: 2023-11-18 09:22:12 INFO     ReqId 139648370206400 Challenge solved!
nov 18 09:22:12 jonathan flaresolverr[336328]: 2023-11-18 09:22:12 DEBUG    ReqId 139648604161728 A used instance of webdriver has been destroyed
nov 18 09:22:12 jonathan flaresolverr[336328]: 2023-11-18 09:22:12 DEBUG    ReqId 139648604161728 Response => POST /v1 body: {'status': 'ok', 'message': 'Challenge solved!', 'solution': {'url': 'https://idope.s>
nov 18 09:22:12 jonathan flaresolverr[336328]: 2023-11-18 09:22:12 INFO     ReqId 139648604161728 Response in 51.614 s
nov 18 09:22:12 jonathan flaresolverr[336328]: 2023-11-18 09:22:12 INFO     ReqId 139648604161728 127.0.0.1 POST http://localhost:8191/v1 200 OK
nov 18 09:22:22 jonathan flaresolverr[336328]: 2023-11-18 09:22:22 INFO     ReqId 139648595752640 Incoming request => POST /v1 body: {'maxTimeout': 60000, 'cmd': 'request.get', 'url': 'https://idope.se/browse.h>
nov 18 09:22:22 jonathan flaresolverr[336328]: 2023-11-18 09:22:22 DEBUG    ReqId 139648595752640 Launching web browser...
nov 18 09:22:22 jonathan flaresolverr[336328]: 2023-11-18 09:22:22 DEBUG    ReqId 139648595752640 Started executable: `/home/jonathan/.local/share/undetected_chromedriver/chromedriver` in a child process with p>
nov 18 09:22:23 jonathan flaresolverr[336328]: 2023-11-18 09:22:23 DEBUG    ReqId 139648595752640 New instance of webdriver has been created to perform the request
nov 18 09:22:23 jonathan flaresolverr[336328]: 2023-11-18 09:22:23 DEBUG    ReqId 139648370206400 Navigating to... https://idope.se/browse.html


[Unit]
Description=flaresolverr
After=network-online.target

[Service]
SyslogIdentifier=flaresolverr
Restart=always
RestartSec=5
Type=simple
User=jonathan
Group=jonathan
Environment="LOG_LEVEL=debug"
Environment="CAPTCHA_SOLVER=none"
WorkingDirectory=/home/jonathan
ExecStart=/home/jonathan/flaresolverr/flaresolverr
TimeoutStopSec=30

[Install]
WantedBy=multi-user.target

**Comment by ilike2burnthing** [comment]
Extend the timeout for FlareSolverr to 120s in Prowlarr's settings. You'll need to enable showing advanced settings, if you haven't already.

**Comment by thegrunge36** [comment]
> Extend the timeout for FlareSolverr to 120s in Prowlarr's settings. You'll need to enable showing advanced settings, if you haven't already.

I had already tried to mount it to 120 in prowlar, but to no effect. For now I'm going to assume that it may be because of the update I did to Debian 12. It seems to be an isolated case. I will continue with docker it does the job.

**Comment by hebval** [comment]
Hi guys, I had the same issue here, but on Windows.

My DEBUG logs for FlareSolverr couldn't find the FlareSolverr checkbox as mentionned above.

When I set my HEADLESS to false, I was just seeing Chromium being opened, browsing to YGG, ticking the box, and not letting the website fully load, and loop these steps over and over until the timeout. Chromium was appearing in FRENCH.

When I set LANG=en_GB, then restart FlareSolverr, there is no issue, everything is working well.

Hope it will help. :)
